content/docs/supportmatrix/_index.md

```diff
@@ -39,14 +39,15 @@ weight: 1
 {{<table "table table-striped table-bordered table-sm">}}
 | Version     | Capability                                                                                            | PowerStore | PowerScale | PowerFlex | PowerMax | Unity XT |
 |-------------|-------------------------------------------------------------------------------------------------------| :--------: | :--------: | :-------: | :------: | :------: |
-| 4.17 - 4.18 | <div style="text-align: left"> [Storage](https://github.com/kiagnose/kubevirt-storage-checkup) </div> | ✅         | ✅        | ✅        | ✅      | ❌      |
-| 4.17 - 4.18 | <div style="text-align: left">  Observability        </div>                                           | ❌         | ✅        | ✅        | ✅      | ❌      |
-| 4.17 - 4.18 | <div style="text-align: left"> Authorization - v2.x  </div>                                           | ❌         | ✅        | ✅        | ✅      | ❌      |
-| 4.17 - 4.18 | <div style="text-align: left"> Resiliency            </div>                                           | ✅         | ✅        | ✅        | ✅      | ❌      |
-| 4.18        | <div style="text-align: left"> Replication (Metro)	</div>                                            | ✅         | ❌        | ❌        | ✅      | ❌      |
+| 4.18 - 4.19  | <div style="text-align: left"> [Storage](https://github.com/kiagnose/kubevirt-storage-checkup) </div>| ✅         | ❌        | ✅        | ✅      | ❌       |
+| 4.18 - 4.19  | <div style="text-align: left">  Observability        </div>                                          | ❌         | ❌        | ✅        | ✅      | ❌       |
+| 4.18 - 4.19  | <div style="text-align: left"> Authorization - v2.x  </div>                                          | ✅         | ❌        | ✅        | ✅      | ❌       |
+| 4.18 - 4.19  | <div style="text-align: left"> Resiliency            </div>                                          | ✅         | ❌        | ✅        | ✅      | ❌       |
+| 4.18 - 4.19  | <div style="text-align: left"> Replication (Metro)	</div>                                            | ✅         | ❌        | ❌        | ✅      | ❌       |
 {{</table>}}
 
-> Note: Replication with Sync/Async modes is not supported.
+> Note: Replication with Sync/Async modes is not supported.<br>
+> Note: For details on the limitations of PowerStore Metro support, please refer [here](../concepts/replication/high-availability/powerstore-metro/#limitations).
```

Comment:
Can we be explicit instead of linking to another place ?

We do not support VM Clone or VM snapshot.